dev-tools/packaging/packages.yml

```diff
@@ -61,7 +61,7 @@ shared:
       /var/lib/{{.BeatName}}/data/{{.BeatName}}-{{ commit_short }}/{{.BeatName}}{{.BinaryExt}}:
         source: build/golang-crossbuild/{{.BeatName}}-{{.GOOS}}-{{.Platform.Arch}}{{.BinaryExt}}
         mode: 0755
-      /var/lib/{{.BeatName}}/data/{{.BeatName}}-{{ commit_short }}/.package.version:
+      /var/lib/{{.BeatName}}/data/{{.BeatName}}-{{ commit_short }}/elastic-agent.package.version:
```

Comment:
Nit: why not just name it `package.version`.  Aren't all files directly under the versioned home folder about Elastic Agent?

Reply:
I have no preference on the name... `package.version` will work just as well since this will be side by side with the executable